assets/js/modules/analytics-4/datastore/webdatastreams.js

```diff
@@ -597,6 +609,36 @@ const baseSelectors = {
 				);
 			}
 	),
+
+	/**
+	 * Checks if a web data stream with the same name already exists.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state             Data store's state.
+	 * @param {string} propertyID        GA4 property ID.
+	 * @param {string} webDataStreamName Web data stream name.
+	 * @return {(boolean|undefined)} TRUE if web data stream already exists; `undefined` if web data streams are not loaded.
```

Comment:
```suggestion
	 * @return {(boolean|undefined)} TRUE if web data stream already exists, FALSE otherwise. Undefined if web data streams are not loaded yet.
```

Reply:
Updated, thanks!